### DISCUSSION TITLE
Vertical wall problem

### DISCUSSION THREAD

**Discussion by webbbn** [supermerill/SuperSlicer#1760] [General]
I'm tryin to switch to SuperSlicer, and really like it so far. I've gotten in calibrated to my printers/filament, and it's working very well, except for one case that I'm stumped on.

I have a couple of prints that print absolutely perfectly, except for a section of vertical surface that extends above the rest of the print.

Assuming it uploads correctly, the picture below is one of them. It printed perfectly up to the top of the main structure, then it printed mostly air for the vertical portion to the left. I've had two or three other prints that did similar things, although not quite as obvious. In those cases it just seemed like the flow was too low, so the structure was very weak.

![image](https://user-images.githubusercontent.com/810831/139622295-3a345d02-4853-4c51-a083-c488404c9eab.png)

I've been blaming my printer until this latest print, but this time it's so clean up until the it tops the top of the majority of the print that I have to think that there's some setting that I don't have set correctly that's making the print fail at that point.

The following shows the print from the model shown above:

![PXL_20211101_044031161](https://user-images.githubusercontent.com/810831/139622743-370b4948-350f-4dae-93ef-dfdb269121af.jpg)

Does anyone have a suggestion for what I might be doing wrong? I'm trying to increase the thickness of the vertical wall, but I don't see why it would fail to print with a 2mm thickness.

**Comment by supermerill** [comment]
your project file?

**Comment by webbbn** [comment]
I didn't save the project file, but I can re-generate it when I get back to my computer.

I think I have an idea what might be happening. I have ironing turned on, so I think it's ironing the whole top side, and then starting to print the vertical wall piece. It's possible that the nozzle/hotend gets gummed up during the ironing process and so the flow is reduced when it restarts printing.

This evening I'm going to try printing it again with ironing turned off to see if that will work.

**Reply by supermerill** [reply]
Totally possible. The flow of the second of ironing is very low, I set the highest value I could, but every printer is different. Also, retraction length can amplify the issue. I think I have an issue about that.

Try without ironing.

**Comment by webbbn** [comment]
I turned off ironing and it printed perfectly, so that was the cause. Thanks for the help!
